ironfish/src/utils/time.ts

```diff
@@ -58,6 +68,29 @@ const renderSpan = (
 
   const parts = []
   let magnitude = 0
+  // Do we want to include years? Is it ok to estimate at 365 days?
```

Comment:
Yes I think it's ok to include years and to estimate at 365 days

Reply:
Update year calculation to use 365 days.